pkg/ebpftracer/c/headers/file_access.h

```diff
@@ -27,9 +18,6 @@ typedef struct file_access_config {
 struct file_access_key {
     u64 cgroup_id;
     u64 inode;
-    u64 pid_start_time;
```

Comment:
I suggest for now go with per container file access. If process is executed multiple times (curl) it still can produce quite a lot reads to load shared libraries. On my local testing removing pid grouping reduces the amount of files tracked more than double.